cmd/dependencyverifier/.gitignore

```diff
@@ -0,0 +1,4 @@
+# General
+.DS_Store
```

Comment:
We already have this `**/.DS_Store` in .gitignore.

I prefer to remove this file here.

Reply:
> We already have this `**/.DS_Store` in .gitignore.
> 
> I prefer to remove this file here.

Thanks for your review. Remove it done.